content/mess.js

```diff
@@ -35,4 +45,15 @@ function mess(c, wait_ms = 2000) {
 
     if (window.mess_resize) mess_resize();
   });
-}
+
+  function createToggle(){
+    const toggleCont = document.createElement('div');
+    toggleCont.classList.add("comp-form-toggle");
+    toggleCont.insertAdjacentHTML('beforeend', `
```

Comment:
you can probably include the div in the string here reducing this down to a one liner.